addons/koji/common/src/main/java/org/commonjava/indy/koji/content/KojiContentManagerDecorator.java

```diff
@@ -584,6 +591,21 @@ private String getPatternString( ArtifactRef artifact, KojiArchiveInfo a )
         return pattern;
     }
 
+    private String getMetaString( ArtifactRef artifact )
+    {
+        String gId = artifact.getGroupId();
+        String artiId = artifact.getArtifactId();
+
+        if ( gId == null || artiId == null )
+        {
+            logger.trace( "Meta ignored, gId: {}, artiId: {}", gId, artiId );
+            return null;
+        }
+        String meta = gId.replace( '.', '/' ) + "/" + artiId + "/" + METADATA_NAME;
```

Comment:
FYI: https://github.com/commonjava/galley/blob/000507a63ddc3ecdff0cccea293dab0de2b9be5d/maven/src/main/java/org/commonjava/maven/galley/maven/util/ArtifactPathUtils.java#L33-L33

Reply:
Ok. Use it.